libkbfs/folder_block_manager.go

```diff
@@ -186,6 +208,30 @@ func (fbm *folderBlockManager) enqueueBlockToDelete(toDelete blocksToDelete) {
 	fbm.blocksToDeleteChan <- toDelete
 }
 
+// enqueueBlockToDeleteNoWait enqueues a block to be deleted just like
```

Comment:
"a block" -> "blocks".

Actually, the function names of this and the one above should be "Block" -> "Blocks" as well.